axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -350,42 +664,72 @@ class TestEvolvedFSM16Noise05(TestFSMPlayer):
         'manipulates_state': False
     }
 
+    """
+    transitions = [
+            (0, C, 8, C),
+            (0, D, 3, D),
+            (1, C, 13, C),
+            (1, D, 15, D),
+            (2, C, 12, C),
+            (2, D, 3, D),
+            (3, C, 10, C),
+            (3, D, 3, D),
+            (4, C, 5, D),
+            (4, D, 4, D),
+            (5, C, 4, D),
+            (5, D, 10, D),
+            (6, C, 8, C),
+            (6, D, 6, D),
+            # (7, C, 5, D),  FSM created by ML algorithm never calls
+            # (7, D, 15, C), states 7 or 9
+            (8, C, 2, C),
+            (8, D, 4, D),
+            # (9, C, 15, D),
```

Comment:
I'm hoping to work through this properly today, can these commented out states simply be removed from the docstring or do we need them here for completeness?